### PR TITLE
Jac design fixes

### PR DESCRIPTION
**Pull request by scabbiaza**

### CODE REVIEW

File: frontend/app/common/components/loading.js [hunk 1]
```diff
@@ -7,10 +7,8 @@ let Loading = React.createClass({
   render() {
     return (
       <DocumentTitle title="Loading...">
-        <div>
-          <section className="container">
-            <i className="fa fa-cog fa-spin"></i>
-          </section>
+        <div id="loading">
+          <i className="fa fa-cog fa-spin"></i>
```
Comment (ivan-kleshnin) [inline frontend/app/common/components/loading.js]: Why `id="loading"`. There may be more than one...

File: bin/fixes/gulp.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-# Replace gulp interpeter to babel-node
+# Replace gulp interpeter to 6to5-node
```
Comment (ivan-kleshnin) [inline bin/fixes/gulp.sh]: This is wrong line. Should be `babel-node`

File: frontend/app/robot/components/add.js [hunk 3]
```diff
@@ -66,23 +66,30 @@ let Add = React.createClass({
...
-                  <TextInput label="Manufacturer" placeholder="Manufacturer" id="model.manufacturer" form={this}/>
-                </fieldset>
-                <div className="buttons">
-                  <button className="btn" type="button" onClick={this.handleReset}>Reset</button>
-                  <button className="btn" type="submit">Submit</button>
+            <section className="container margin-top-lg">
+              <div className="row">
+                <div className="col-xs-12 col-sm-3">
+                  <div className="thumbnail thumbnail-robot">
+                    // loading by id created an error, removed this img
```
Comment (ivan-kleshnin) [inline frontend/app/robot/components/add.js]: Don't understand this

Comment (scabbiaza) [inline frontend/app/robot/components/add.js]: I had error in add component with loading photo.
But we decided we will not have photo here. Will remove this.

**Comment by scabbiaza** [comment]
Pls review one more time

**Pull request merged**